tests/test_engine/test_model_traffic_capture.py

```
@@ -153,6 +176,118 @@ def _capture_response_body(
+                headers={"content-type": "application/json"},
+                latency_ms=10.0,
+                ctx=ctx,
+                body=_capture_response_body(),
+            )
+        )
+
+        # The eval-loop error paths drain the session and drop the return value
+        # without iterating it, so only __del__ -> discard() unlinks the spool file.
+        drained = store.drain_session(session_id)
+        drain_path = drained._path
+        assert drain_path is not None and drain_path.exists()
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)